hubspot3/test/test_deals.py

```diff
@@ -1,14 +1,25 @@
 """
-testing hubspot3.deals
+Testing hubspot3.deals
+This file tests the methods in the hubspot3.deals file
 """
 import pytest
 from hubspot3.deals import DealsClient
 from hubspot3.error import HubspotNotFound, HubspotBadRequest
 from hubspot3.test.globals import TEST_KEY
-
+from hubspot3 import deals
+import json
+from unittest.mock import Mock
```

Comment:
Aqui acho que a estrutura ideal dos imports seria

```
import json
from unittest.mock import Mock

from hubspot3 import deals
from hubspot3.error import HubspotNotFound, HubspotBadRequest
from hubspot3.test.globals import TEST_KEY

import pytest
```
A ordenação aqui segue o padrão [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) de imports.
E acredito que você pode remover o `from hubspot3.deals import DealsClient` se você está fazendo `from hubspot3 import deals`. Aí você usa `deals.DealsClient` quando for chamar no código (vou criar outra sugestão).